textflip/commandline.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+import sys
+
+from .rotation import flip
+
+
+def flip_cli():
+    if not sys.stdin.isatty():
+        lines = input().splitlines()
+        for line in lines:
+            print(flip(line))
+
+        return
+
+    parser = argparse.ArgumentParser()
```

Comment:
I think `argparse` is overkill when all you do is take the first word, a.k.a. `sys.argv[1]`.

Reply:
argparse was a pretty straightforward way of getting a sensible error message in case you run the command without any text. but i guess you can add the error manually, something like

```py
if len(sys.argv) == 0:
    sys.stderr.write("Error: no text provided")
    sys.exit(1)
```

would that be better?

Reply:
Honestly, no: In that case I would expect it to wait for input (like `cat`). I would therefore reverse the logic: If there are arguments, flip them. Otherwise, flip stdin.